misc/services/mobile-webui/mobile-webui-frontend/src/reducers/wfProcesses/manufacturing.js

```diff
@@ -0,0 +1,17 @@
+import { toQRCodeString } from '../../utils/qrCode/hu';
+import { getLineByIdFromActivity, getStepsArrayFromLine } from './index';
+
+export const getNonIssuedStepByQRCodeFromActivity = (activity, lineId, qrCode) => {
```

Comment:
for the future, when having > 1 params, I would really prefer to use `{` `}` params